Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1554,6 +1554,7 @@ def lhegensim(fragment,howMuch):
 defaultDataSets={}
 defaultDataSets['2017']='CMSSW_8_1_0_pre5-80X_upgrade2017_realistic_v4_resubUPG17-v'
 defaultDataSets['2023GReco']='CMSSW_8_1_0_pre4-80X_mcRun2_asymptotic_v13_2023LReco-v'
+defaultDataSets['2023LReco']='CMSSW_8_1_0_pre4-80X_mcRun2_asymptotic_v13_2023LReco-v'
 defaultDataSets['2023tilted']='CMSSW_8_1_0_pre4-80X_mcRun2_asymptotic_v13_2023tilted-v'
```

Comment:
I'm not sure how these are supposed to be kept up to date. The GReco and tilted geometries are changing in this PR, so obviously there is no appropriate MinBias available yet for the PU workflows... @boudoul?